iac-extensions/terraform/src/main/java/org/sonar/iac/terraform/checks/gcp/PublicAccessCheck.java

```diff
@@ -71,8 +80,25 @@ protected void registerResourceConsumer() {
       resource -> resource.attribute("visibility")
         .reportIf(equalTo("public"), MESSAGE)
         .reportIfAbsent(OMITTING_DNS));
-  }
 
+    register("google_container_cluster",
+      resource -> {
+        BlockSymbol config = resource.block("private_cluster_config");
+        config.reportIfAbsent(OMITTING_KUBERNETES);
+        AttributeSymbol nodes = config.attribute("enable_private_nodes");
+        AttributeSymbol endpoint = config.attribute("enable_private_endpoint");
+
+        if (!(nodes.isPresent() || endpoint.isPresent())) {
```

Comment:
Why not have have also an `isAbsent()` method and rewrite this as
`if (nodes.isAbsent() && endpoint.isAbsent())` // for the sake of readability